docs/en/advanced_guides/data_flow.md

```diff
@@ -1 +1,57 @@
-# Data Flow
+# Dataflow
+
+In this chapter, we will introduce the dataflow and data format convention between the internal modules managed by the [Runner](https://mmengine.readthedocs.io/en/latest/tutorials/runner.html).
+
+## Overview of dataflow
+
+As illustrated in the [Runner document of MMEngine](https://mmengine.readthedocs.io/en/latest/tutorials/runner.html), the following diagram shows the basic dataflow.
+
+![Basic dataflow](https://user-images.githubusercontent.com/112053249/199228350-5f80699e-7fd2-4b4c-ac32-0b16b1922c2e.png)
+
```

Comment:
Add detail explanation for red/blue/green lines. Just like what [MMEngine](https://github.com/open-mmlab/mmengine/blob/main/docs/en/tutorials/runner.md) did.

![image](https://user-images.githubusercontent.com/24582831/208358140-7d3e7071-938b-46bf-8607-45da849c04fc.png)